packs/st2/actions/chatops_format_list_result.py

```diff
@@ -0,0 +1,34 @@
+from st2actions.runners.pythonrunner import Action
+
+from prettytable import PrettyTable
+
+__all__ = [
+    'St2ChatOpsFormatListResult'
+]
+
+
+class St2ChatOpsFormatListResult(Action):
```

Comment:
Nice. This is similar to what I was thinking with formatting. The only downside is that its another action to handle this, as opposed to something the client handles.